immuneML/reports/Report.py

```diff
@@ -6,10 +6,12 @@
 
 class Report(metaclass=abc.ABCMeta):
```

Comment:
could you not change this file? :D

Reply:
Sure I can, I changed it because I got problems in my interpreter because the lines were too long. It is just to me?

Reply:
as the link you posted (https://docs.immuneml.uio.no/_modules/source/reports/ml_reports/Coefficients.html#Coefficients)
the text is cut in my browser and I have to navigate in the bar to read.